services/markup/processorhelper_codepreview_test.go

```diff
@@ -63,7 +63,7 @@ func TestProcessorHelperCodePreview(t *testing.T) {
 	<table class="file-view">
 		<tbody><tr>
 				<td class="lines-num"><span data-line-number="1"></span></td>
-				<td class="lines-code chroma"><code class="code-inner"><span class="gh"># repo1</code></td>
+				<td class="lines-code chroma"><span class="code-inner"><span class="gh"># repo1</span></td>
```

Comment:
Why `span` instead of `div`?

Reply:
Because "span" doesn't work ....

![image](https://github.com/go-gitea/gitea/assets/2114189/11ddf002-cb9f-4158-9b54-0876ad3eace8)